src/state_title_screen.c

```diff
@@ -409,6 +409,14 @@ void title_screen_draw_logo(f32 arg0) {
     gDPPipeSync(gMasterGfxPos++);
 }
 
+#if VERSION_CN
+#define VAR_1 20
```

Comment:
I'm not really sure what these variables mean. Based on the docs `VAR_2` would be `yEnd`, but I don't understand in what range they work. Is `(0, 0)` top left, and `(1024, 1024)` bottom right?

Reply:
Could use some help with these names

Reply:
no shame in having bad names in the short term. I don't think this should block the PR from being merged. but to answer your question, 0 0 is top left, yeah.